app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx

```diff
@@ -6,17 +6,18 @@ import Form from "./Form";
 
 import { NOTES_FORM_INITIAL_FORM_VALUES } from "../constants";
 
-const NewNotePane = ({ showPane, setShowPane }) => {
+const NewNotePane = ({ showPane, setShowPane, createNote }) => {
```

Comment:
Keep the React component name and the filename same.

Reply:
Sorry I missed that, `NewNotePane` was part of the base [`wheel code`](https://github.com/bigbinary/wheel/blob/fb44f4f583d0fa4de9d69f853c33c4f963a1aa38/app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx#L9).  I will modify it in the upcoming PR.